### ISSUE TITLE
Make new editor compatible with Enriched Editor plugin (or vice versa)

### ISSUE THREAD

**Issue by EvoRay** [ClassicPress-research/ClassicPress-Editor#58]
I will also open a feature request on Enriched Editor if I can find it on Github.

**Comment by johnalarcon** [comment]
I agree. If users are going to lose all their WYSIWYG functionality, there's no good reason for them to switch...or to even build a new editor.

**Comment by joyously** [comment]
I've only scratched the surface, but could you try it with #62 ?

**Comment by johnalarcon** [comment]
I tried Enriched Editor (EE) on #62. The admin interface has some slightly broken CSS, but, it appears to work for controlling the toolbar. I successfully added and removed buttons, moved them around to new positions, populated the 3rd and 4th rows, show/hide the menu bar. I didn't try all the EE settings; mostly, this was to test how the new editor would react to the EE builder's config settings. It's not that far off. There were a few icons missing and a few "bomb" (presumably placeholder) icons in other places (although after a page refresh, I can't find any of the bombs now.)

#### Enriched Editor Settings
![image](https://user-images.githubusercontent.com/12477319/133945390-4ba1927b-50b0-4a85-b839-bd010dab1396.png)

The image above depicts the broken CSS for the menu bar display and in the Unused Buttons section, some icons are showing, some are not. Notably, the Unused Buttons, when dragged into the builder, do place the actual button there, although it's invisible with the missing icon – however, in the actual editor, the button does show with an icon.

#### ClassicPress Editor operating on Enriched Editor's Settings
![image](https://user-images.githubusercontent.com/12477319/133945401-cf5a2c8e-a709-41e8-bf49-b8e7e6e30281.png)
